pkgs/build-support/testers/lychee.nix

```diff
@@ -0,0 +1,49 @@
+deps@{ formats, lib, lychee, stdenv, writeShellApplication }:
+let
+
+  # See https://nixos.org/manual/nixpkgs/unstable/#tester-lycheeLinkCheck
+  # or doc/builders/testers.chapter.md
+  lycheeLinkCheck = {
+    site,
+    remapUrl ? null,
```

Comment:
```suggestion
```

Reply:
I've replaced it by the `remap` argument, which is more general, correctly documented (regex!) and about as easy to use for the original use case of `remapUrl` but without catching users in an overly specific solution.